snuba/settings_base.py

```diff
@@ -43,6 +43,7 @@
 # Snuba Options
 
 SNAPSHOT_LOAD_PRODUCT = 'snuba'
+BULK_CLICKHOUSE_BUFFER = 1000
```

Comment:
Was there any decision process behind this number as the default value? Or is this just a placeholder?

Reply:
just a place holder.